manifests/mod/event.pp

```diff
@@ -47,18 +47,18 @@
 # @see https://httpd.apache.org/docs/current/mod/event.html for additional documentation.
 # @note Unsupported platforms: SLES: all
 class apache::mod::event (
-  $startservers           = '2',
-  $maxclients             = '150',
-  $maxrequestworkers      = undef,
-  $minsparethreads        = '25',
-  $maxsparethreads        = '75',
-  $threadsperchild        = '25',
-  $maxrequestsperchild    = '0',
-  $maxconnectionsperchild = undef,
-  $serverlimit            = '25',
-  $apache_version         = undef,
-  $threadlimit            = '64',
-  $listenbacklog          = '511',
+  Variant[String,Boolean] $startservers                       = '2',
+  Variant[String,Boolean] $maxclients                         = '150',
+  Optional[Variant[String,Boolean]] $maxrequestworkers        = undef,
+  Variant[String,Boolean] $minsparethreads                    = '25',
+  Variant[String,Boolean] $maxsparethreads                    = '75',
+  Variant[String,Boolean] $threadsperchild                    = '25',
+  Variant[String,Boolean] $maxrequestsperchild                = '0',
+  Optional[Variant[String,Boolean]] $maxconnectionsperchild   = undef,
+  Variant[String,Boolean] $serverlimit                        = '25',
```

Comment:
In what case would `serverlimit` be a boolean? Would it make more sense to be `Variant[String, Int]`?